packages/edit-navigation/src/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	registerCoreBlocks,
+	__experimentalRegisterExperimentalCoreBlocks,
+} from '@wordpress/block-library';
+import { render } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Layout from './components/layout';
+
+export function initialize( id, settings ) {
+	registerCoreBlocks();
```

Comment:
Nitpick, do we need all these? We can probably get away with only registering the Nav and Nav Link blocks.